src/vs/base/browser/dom.ts

```diff
@@ -820,11 +820,20 @@ export function hasParentWithClass(node: HTMLElement, clazz: string, stopAtClazz
 	return !!findParentWithClass(node, clazz, stopAtClazzOrNode);
 }
 
-export function createStyleSheet(container: HTMLElement = document.getElementsByTagName('head')[0]): HTMLStyleElement {
+export function createStyleSheet(container: HTMLElement | null = null): HTMLStyleElement {
+	if (!container) {
+		if ((window as any).monacoShadowRoot) {
```

Comment:
There can be multiple editors instantiated in a single page, so using a global here is not possible.

Reply:
I have reworked this, such that all calls to createStyleSheet pass in a relevant container in the shadow dom case.